syntax = "proto3";

package skaimsg.globaltrack;

message GlobalTrackProtoMsg {
    repeated float val = 1;
}